### PR TITLE
pump/storage: Random cleanups

### PR DESCRIPTION
**Pull request by suzaku**
### What problem does this PR solve? 

Clean up come code in pump/storage.

### What is changed and how it works?

1. Use predefined constants in leveldb/opt to improve readability of
   leveldb config values.
1. Use path/filepath and strings functions to simplify path manipulation
1. Rename confusing variable

### Check List 

Tests 

 - Unit test

Code changes

Side effects

Related changes

### CODE REVIEW

File: pump/storage/storage.go [hunk 1]
```diff
@@ -732,26 +732,26 @@ func (a *Append) writeToValueLog(reqs chan *request) chan *request {
 		var bufReqs []*request
 		var size int
 
-		write := func(reqs []*request) {
-			if len(reqs) == 0 {
+		write := func(batch []*request) {
```
Comment (WangXiangUSTC) [inline pump/storage/storage.go]: the value's type is request, maybe reqs is better, I think don't need change value name

Comment (suzaku) [reply pump/storage/storage.go]: The name `reqs` is already taken above for the channel.

**Comment by GregoryIan** [comment]
LGTM

**Pull request merged**